blast_config/derived.c

```diff
@@ -518,6 +518,200 @@ derived_tng_t derived_list[] = {
     LINCOM("Inner_frame_voltage_3", "INNER_FRAME_VM_3", 10, 0),
 
     // current monitoring
+    LINCOM("Current_fc1", "current_fc1", 5, CURR_LOOP_OFFSET),
```

Comment:
These two blocks look like duplicates of the ones below the thermistor lookup table sections.